chain/README-dev.md

```diff
@@ -1,39 +1,114 @@
 # Developer Documentation
 
+## Branches
+
+We use `laika-node/pre-release`/`tlbc-node/pre-release` for release candidates
+and `laika-node/release`/`tlbc-node/release` for releases. We only merge from
+`master` to `*/pre-release`, from `*/pre-release` to `*/release` and from
+`*/release` back to `master`.
+
+The version on the master branch should always be a `dev` version. The
+`*/pre-release` branches should always have a `rc` version and on `*/release`
+branches we only have `prod` releases (where `prod` doesn't appear in the
+version number, i.e. the version number is always `major.minor.patch`).
+
+## bumpversion
+
+Both chains use [bumpversion](https://pypi.org/project/bumpversion/) in order to
+maintain their version. bumpversion can be used to automatically upgrade the
+current version in the respective `VERSION` file within the `chain/laika` and
+`chain/tlbc` directories.
+
+bumpversion is configured via the `.bumpversion.cfg` configuration files in
+`chain/laika` and `chain/tlbc`. If you want to run bumpversion to change
+a chain's version, please change to the chain's directory first. Note that
+bumping the version will automatically create a new commit.
+
+## Making A Pre-release
+
+Adopt the following instructions for the respective chain (demonstrated for
+the Laika test network) new version:
+
+```sh
+$ git checkout master
+$ git pull
+$ git checkout -b laika-node/prepare/pre-release-<version>
+$ cd chain/laika
+$ bumpversion release
+```
+
+and open a PR for this new branch to `laika-node/pre-release`.
+
+The version in the `*/pre-release` branches should always be a 'release candidate'
+version with the `rc` suffix.
+
+## Making A Release
+
+Run the following commands and adopt for the respective chain (demonstrated for
+the Laika test network) and new version:
+
+```sh
+$ git checkout -B laika-node/pre-release --track origin/laika-node/pre-release
+$ git pull
+$ git checkout -b laika-node/prepare/release-<version>
+$ cd chain/laika
+$ bumpversion release
+```
+
+and open a PR for this new branch to `laika-node/release`.
+
+The version in the `*/release` branches should always be a 'release' version
+with a clean version number.
+
+## Merge Release Back
+
+After a release has been made, it should been merged back to the `master`
+branch. Adopt the following instructions for the respective chain (demonstrated
+for the Laika test network) and the next version:
+
+```sh
+$ git checkout -B laika-node/release --track origin/laika-node/release
+$ git pull
+$ git checkout -b laika-node/prepare/master-<next-version>
+$ cd chain/laika
+$ bumpversion path
+$ git checkout master
+$ git pull
+$ git merge -m 'Merge with laika-node/release branch' laika-node/prepare/master-<next-version>
```

Comment:
Any reason why you want to do this without a PR?

Reply:
I changed to the same as for all other stages. :+1: